### DISCUSSION TITLE
I want to know how to export file without filename using pandoc

### DISCUSSION THREAD

**Discussion by Juan-hwt** [jgm/pandoc#10337] [General]
I use obsidian to write docs, like this
![image](https://github.com/user-attachments/assets/afefecd6-3770-48f1-b2f8-e8de9f58f4d2)

when using pandoc, export docx, the resault like this
![image](https://github.com/user-attachments/assets/9fbbec0f-5e88-4c17-a393-7980a1849f31)

I don't want the filename to export, how can I deal with it ?

anyone help me?

**Comment by alerque** [comment]
This sounds like a question for Obsidian. Pandoc offers lots of ways to control this but it isn't clear how you are calling it. Are you calling Pandoc yourself? In that case you could easily change the options about including headers/title/metadata/filtering/whatever. But since you say "export" I assume you mean from Obsidian, in which case Obsidian is probably calling Pandoc so it's really up to Obsidian to expose the right options for you.

**Reply by Juan-hwt** [reply]
I must to say thanks to you fist, and I have setted obsidian like this :

--reference-doc=D:\xxx.docx

I just don't know how to change the settings, exclude filename like above! who can help me?

I'm confused with the documents about https://pandoc.org/MANUAL.html, nothing give me advise, so, come here to find some help

**Comment by badumont** [comment]
Could you please show the Markdown file instead of a screenshot of the app? It will help determine how Obsidian represents the file title in Markdown and, maybe, how to remove it from the exported file.

**Reply by Juan-hwt** [reply]
[I don't want this filename in word(docx).md](https://github.com/user-attachments/files/17534064/I.don.t.want.this.filename.in.word.docx.md)

here is the markdown file，thank you

**Comment by badumont** [comment]
The text that you don't want to get exported is encoded as a level-1 heading. It's a bit weird not to want a heading to be exported, like wanting *this* not to be italicized. If you don't want this string in the export, you can format it as a comment (https://help.obsidian.md/Editing+and+formatting/Basic+formatting+syntax#Comments).

**Reply by Juan-hwt** [reply]
[I don't want this filename in word(docx).md](https://github.com/user-attachments/files/17539194/I.don.t.want.this.filename.in.word.docx.md)
I think you did't get the problem, you can try the markdown file above, as you have said
all will happen, nothing change

**Reply by alerque** [reply]
Write a Lua filter that processes `Header` and checks if the level is 1, and returns an empty value if it is. Optionally change all the other heading levels so 2 becomes 1, 3 becomes 2, etc. if that works with your output handling.

**Reply by Juan-hwt** [reply]
lua…… I don't know how to write, can you give me an example?

**Comment by badumont** [comment]
I understand that you introduce stuff you don't want to be output as level-1 headers, which is not what headers are meant for. Comments are made for that.
